software/contrib/gate_phaser.md

```diff
@@ -0,0 +1,64 @@
+# Gate Phaser
+
+author: Nik Ansell (github.com/gamecat69)
+
+date: May 2024
+
+labels: sequencer, gates
+
+A script which attempts to answer the question "What would Steve Reich do if he had a EuroPi?".
+
+Gates are sent from outputs 1-6 which are offset (or out of phase) with each other. This creates a group of gates that drift in and out of phase with each other over time.
+
+You can use this script to create Piano Phase type patches, dynamic rhythms which evolve over time and eventually resolve back in phase, or ... well lots of other things that would benefit from having gates which are delayed from each other.
+
+# Inputs, Outputs and Controls
+
+![Operating Diagram](./gate_phaser-docs/gate_phaser.png)
```

Comment:
In addition to the diagram, a text summary of the inputs & outputs is helpful. If you just open the readme in a text editor all of the information needed should be there, without needing to open an image.

A summary of the ranges of the inputs and the list of multiples that can be cycled through would also be good to include.